intelmq/lib/upgrades.py

```diff
@@ -887,7 +900,7 @@ def v310_feed_changes(configuration, harmonization, dry_run, **kwargs):
     ((3, 0, 1), (v301_deprecations,)),
     ((3, 0, 2), ()),
     ((3, 1, 0), (v310_feed_changes, v310_shadowserver_feednames)),
-    ((3, 2, 0), ()),
+    ((3, 2, 0), (v320_update_turris_greylist_url)),
```

Comment:
That line misses a trailing comma to make it a tuple:

```
    ((3, 2, 0), (v320_update_turris_greylist_url, )),
```
Upgrading to version 3.2.0.
  File "/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/intelmq/bin/intelmqsetup.py", line 184, in intelmqsetup_core
    controller.upgrade_conf(state_file=state_file, no_backup=True)
  File "/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/intelmq/bin/intelmqctl.py", line 1152, in upgrade_conf
    for function in bunch:
TypeError: 'function' object is not iterable
```

Reply:
Fix of the fix incoming in few...